### DISCUSSION TITLE
Support Email mailto Parameters in Email Field

### DISCUSSION THREAD

**Discussion by besweeet** [pods-framework/pods#6904] [Ideas]
### Problem to Solve

If adding mailto parameters, such as subject, WordPress will complain about either an invalid email address or a missing email altogether.

### Proposed Solution

Have the email field type support ?= mailto parameters.

**Comment by JoryHogeveen** [comment]
Hi @besweeet 

I'm not sure if I get the issue.
How is this related to Pods?
This sounds like something a different plugin should fix, like WP Mail SMTP for example.

Cheers, Jory

**Comment by besweeet** [comment]
> Hi @besweeet
> 
> I'm not sure if I get the issue. How is this related to Pods? This sounds like something a different plugin should fix, like WP Mail SMTP for example.
> 
> Cheers, Jory

I don't know if it's related to Pods or if it's WordPress that's not liking what's being entered into an email field configured by Pods.

For example, I created a new post pod that contains an email field (marked as required). If I enter `emailaddress@gmail.com` into the field, the post can be published with no issue. But, if I instead enter `emailaddress@gmail.com?subject=TestSubject` into the field, the post cannot be saved and an error of "Email is required" is displayed. If I remove the required checkbox and try it again, an error of "Invalid e-mail provided for Email" is displayed.

**Comment by JoryHogeveen** [comment]
Hi @besweeet 

Ah check, that is not a bug bug the expected behavior.
The email field does not support other params, only an email.

If you need to pre-fill subject and/or body, please create separate fields for this and format your own email link with these.

Cheers, Jory

**Reply by besweeet** [reply]
> Hi @besweeet
> 
> Ah check, that is not a bug bug the expected behavior. The email field does not support other params, only an email.
> 
> If you need to pre-fill subject and/or body, please create separate fields for this and format your own email link with these.
> 
> Cheers, Jory

This worked! Thanks. I'm using Elementor, so I used the shortcode link option: [mailto:] [pods field="my_email_field"] [?subject=Test]

**Comment by JoryHogeveen** [comment]
Converted this issue to a discussion to possibly implement it in the future if more users request it.

**Comment by sc0ttkclark** [comment]
This could be useful as an option for display at some point, could be a snippet that would handle that
